beacon-chain/core/blocks/exit.go

```diff
@@ -186,6 +190,18 @@ func verifyExitConditions(validator state.ReadOnlyValidator, currentSlot primiti
 			params.BeaconConfig().ShardCommitteePeriod,
 			validator.ActivationEpoch()+params.BeaconConfig().ShardCommitteePeriod,
 		)
+	} 
+
+	if st.Version() >= version.Electra {
```

Comment:
do we want to break this out from the core blocks into electra?

Reply:
Eventually. Breaking this out would involve breaking the other forks out and it would be a large change. Currently trying to get to devnet-1 while taking note of this. I'll take a note of this, thanks